problems/sevenBoom.js

```diff
@@ -1,11 +1,21 @@
 /** Write a function that returns all the values from 1 to n inclusive, replacing all numbers that are a multiple of seven, or contain seven with "BOOM".
-* Sample Input: 20
-* Sample Output: [1,2,3,4,5,6,'BOOM',8,9,10,11,12,13,'BOOM',15,16,'BOOM',18,19,20]
-* @param {number} n - The number to count up to
-* @returns {number[]} - An array matching the pattern described above
-*/
-function sevenBoom() {
-
+ * Sample Input: 20
+ * Sample Output: [1,2,3,4,5,6,'BOOM',8,9,10,11,12,13,'BOOM',15,16,'BOOM',18,19,20]
+ * @param {number} n - The number to count up to
+ * @returns {number[]} - An array matching the pattern described above
+ */
+function sevenBoom(n) {
+  let output = [];
+  let num = 0;
+  while (num < n) {
+    num++;
+    output.push(num);
+  }
+  for (let i = 0; i < output.length; i++) {
+    if (output[i] % 7 === 0 || output[i] % 10 === 7) {
```

Comment:
This should have been a loop that goes from 0 => n instead of putting 0 => in a separate array and then looping through that array. 

This passes the test cases but how would you alter this to include a number like 75, which has a 7 but doesn't meet your conditions?